compute/sqlserver/powershell/create-sql-instance-availability-group-prerequisites.ps1

```diff
@@ -0,0 +1,384 @@
+﻿#Requires -Version 5
+# Copyright(c) 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+##############################################################################
+#.SYNOPSIS
+# Run the prerequisites before running the script: 
+#   create-sql-instance-availability-group.ps1
+#
+#.DESCRIPTION
+# The script create-sql-instance-availability-group.ps1 assumes you have 
+# custom network with 3 subnetworks. It also expects a Windows AD domain.
+# 
+# This script can be used to create all of those prerequisites
+#
+# You should run this script one section at a time since not all sections
+# may apply in your case
+#
+#.NOTES
+# AUTHOR: Anibal Santiago - @SQLThinker
+##############################################################################
+
+
+################################################################################
+# Specify the parameters
+################################################################################
+# Network configuration
+$network      = 'wsfcnet'      # Name of the Custom network

```

Comment:
We have a document that goes step by step how to create a HA Availability Group. The document references the script create-sql-instance-availability-group.ps1 as a way to automate the steps. But, it also says that before running the script, they need to do some pre-requisites. We realized that some customers are not familiar with GCP and will not know how to do those pre-requisites. That is why we created this "-prerequisites" script for them. It matches the same values we used in our documentation. We expected that it will be used by clients following the instructions document and that will use exactly the same values we used in the doc.